E.Deezer/DeezerClient.cs

```diff
@@ -114,31 +115,31 @@ public Task<IChart> GetChart(ulong aId, uint aStart, uint aCount)
             return GetChart(method, parms, aStart, aCount);
 
         }
-
-        public Task<T> GetPlainWithError<T>(string aMethod) where T : IHasError
+       
+        public Task<T> GetPlainWithError<T>(string aMethod, IList<IRequestParameter> aParams = null) where T : IHasError
         {
-            return iExecutor.ExecuteGet(aMethod, RequestParameter.EmptyList)
+            if (aParams == null) aParams = RequestParameter.EmptyList;
+            return iExecutor.ExecuteGet(aMethod, aParams)
                             .ContinueWith(t =>
                             {
                                 CheckHttpResponse(t);
                                 T deserialized = DeserializeResponse<T>(t.Result.Content).Result;
-
+                                var msg = t.Result.Content.ReadAsStringAsync().Result;
```

Comment:
`msg` doesn't appear to be used. Think that could go!

Reply:
sure, ist was for my debug. :-)